docs/design/live_migration.md

```diff
@@ -0,0 +1,176 @@
+# KubeVirt virtual machine bridge binding pod network seamless live migration
+
+
+## Introduction
+
+At kubevirt every VM is executed inside a "virt-launcher" pod; how the "virt-launcher" pod net interface is "bound" to the VM is specified by the network binding at the VM spec, ovn-kubernetes support live migration of the KubeVirt network bridge binding. 
+
+At KubeVirt bridge binding the virt-launcher interface is replaced by a bridge with one leg on the pod interface and the other at the VM tap device. When IPAM is configured on the pod interface, a DHCP server is started to advertise the pod's IP to DHCP aware VMs.
```

Comment:
This sentence is very confusing.

What are you replacing ? And with what ?...

AFAIU, KubeVirt doesn't replace anything. It adds a bridge and a tap device, and connects the existing pod interface as a port of the aforementioned bridge.